layouts/_default/baseof.html

```diff
@@ -4,7 +4,7 @@
     {{ block "head" . }}{{ partial "site-head.html" .}}{{ end }}
   </head>
 
-  <body class="bg-white">
+  <body class="bg-hexagon bg-center bg-contain">
```

Comment:
Nice done. it's exactly what i did but i wasn't sure about how to handle the other screen size.